demo/external-logging-elasticsearch/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+  elk:
+    image: sebp/elk:es241_l240_k461
+    container_name: elk
+    ports:
+      - "5601:5601" # Kibana
+      - "9200:9200" # Elasticsearch
+      - "5044:5044" # Logstash
+    expose:
+      - "5601"
+      - "9200"
+      - "5044"
+  jenkins:
```

Comment:
Needs an `agent` container as in https://github.com/jenkinsci/logstash-plugin/pull/18. Need not be elastic for starters; the important thing is to prove that the ELK communications from `sh` steps have an origin IP which is not the master.

Reply:
K8S will get adopted whenever people need it, right? If you can do better using K8S, go for it from my PoV.

Reply:
Let's keep it as is. I will do Kubernetes upon request. The POD definition itself is trivial, the only problem is to get K8s running :(